persistent/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for persistent
 
+## 2.11.1.0
```

Comment:
I've *got* to update the changelog to make it less confusing - can you put this in `2.11.0.0`?

Reply:
Sure thing! I’ve seen people use a heading “unreleased changes” at the top of changelogs, which seems to work okay.